workers/social/lib/social/models/computeproviders/stacktemplate.coffee

```diff
@@ -435,7 +435,14 @@ module.exports = class JStackTemplate extends Module
 
             data['meta.modifiedAt'] = new Date
 
-            next()
+            originalId = data.config?.clonedFrom
+            return next()  unless originalId
+
+            # update clonedSum information in the template, if it is exists
+            JStackTemplate.one$ client, { _id: originalId }, (err, template) ->
+              if template and not err
+                data.config.clonedSum = template.template.sum
```

Comment:
Let's say: Original Stack Template = A, cloned stack template = B, Stack A's sum = `xxx1`, Stack B's config's clonedSum = `xxx1`, Stack B's sum = `xxx1` as well.

We made Stack A's sum `xxx2` and then Stack B gets updates and we also update cloned stack, and made its sum `xxx2`. If we don't have above code, its config's clonedSum will remain `xxx1`.If so, it will be `xxx2`

Then we just update the Stack B, and made its sum `xxx3`. If Stack B's config's clonedSum is `xxx1` that means original stack template updated but cloned stack template has no idea about the new sum info.

So either we update the cloned stack template because of the original one updated or just the cloned one updated, we need to update its config's clonedSum info as well, otherwise it will read stale sum info.